projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -425,6 +432,11 @@ function render_block( $attributes ) {
  * @return string the actual post access level (see projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js for the values).
  */
 function get_post_access_level() {
+	if ( ! is_singular() ) {
+		// There is no "actual" current post.
+		return 'everybody';
```

Comment:
```suggestion
		return Token_Subscription_Service::POST_ACCESS_LEVEL_EVERYBODY;
```

Reply:
Added in [5d25c5e](https://github.com/Automattic/jetpack/pull/29860/commits/5d25c5e7225c50581a0412ce382b367d2e535fc7)